src/amg/azext_amg/backup.py

```diff
@@ -189,32 +118,17 @@ def _save_library_panel_setting(panel_name, file_name, library_panel_settings, f
     logger.info("    -> %s", file_path)
 
 
-def _get_individual_library_panel_setting_and_save(panels, folder_path, log_file, grafana_url, http_headers):
-    file_path = folder_path + '/' + log_file
-    if panels:
-        with open(file_path, 'w', encoding="utf8") as f:
-            for panel in panels:
-                panel_uri = panel['uid']
-
-                (status, content) = get_library_panel(panel_uri, grafana_url, http_headers)
-                if status == 200:
-                    _save_library_panel_setting(
-                        panel['name'],
-                        panel_uri,
-                        content['result'],
-                        folder_path)
-                    f.write(panel_uri + '\t' + panel['name'] + '\n')
-
-
 # Save snapshots
 def _save_snapshots(grafana_url, backup_dir, timestamp, http_headers, **kwargs):  # pylint: disable=unused-argument
     folder_path = f'{backup_dir}/snapshots/{timestamp}'
 
     if not os.path.exists(folder_path):
         os.makedirs(folder_path)
 
-    _get_all_snapshots_and_save(folder_path, grafana_url, http_get_headers=http_headers)
-    _print_an_empty_line()
```

Comment:
is there a reason the empty line print was removed?

Reply:
I moved them to backup_core.py

Reply:
I am not seeing a line print in get_all_snapshots

Reply:
Oh, that's true, thanks for catching that!